### DISCUSSION TITLE
Scripts is not execute afte component update

### DISCUSSION THREAD

**Discussion by 0x0pharaoh** [livewire/livewire#6989] [7. Bugs (archived)]
### Livewire version

v3

### Which PHP version are you using?

PHP 8.2

### Additional information

Component :

```
public function render()
{
        return view('livewire.users.index',['users' => User::getUsers(per_page:10)]);
}

public function deactivate($id)
{
        User::toggleStatus($id,User::USER_STATUS['deactive']);
        session()->flash('success','client updated !'); 
}
```

View (html) :

```
@push('scripts')
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
    <script src="{{ asset('js/toastr.min.js') }}"></script>
    <script>
        toastr.options = {
            "closeButton": true,
            "debug": false,
            "newestOnTop": true,
            "progressBar": true,
            "positionClass": "toast-top-right",
            "preventDuplicates": true,
            "onclick": null,
            "showDuration": "300",
            "hideDuration": "1000",
            "timeOut": "5000",
            "extendedTimeOut": "1000",
            "showEasing": "swing",
            "hideEasing": "linear",
            "showMethod": "fadeIn",
            "hideMethod": "fadeOut"
        };
    </script>
@endpush


@if (session('success'))
    <script>toastr.success("{{session('success')}}")</script>
@endif

```

Output : session work but scripts is not execute for same component code 

Note : This same code works in Livewire V2

![bug](https://github.com/livewire/livewire/assets/123071358/4f28279a-b5e6-4526-b540-9399a4b5a6db)

PS : When set session from different component and redirect to then working script executed 

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by Geovanek** [comment]
I'm having some problems since I updated to Livewire 3 to.

We know that with each rendering of components, any *javascript* inserted loses its reactivity.
In Livewire 2, using the
```js
document.addEventListener("livewire:load", () => {
   ...
});
```
the javascript was always "reset" within the component working perfectly.

For example, I had the following code:
```js
document.addEventListener("livewire:load", () => {
         $(function () {
             setTimeout(1000)
             var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
             var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
                 return new bootstrap.Tooltip(tooltipTriggerEl)
             })
         })
});
```
which always reactivated the bootstrap Tooltip function.

However, since I updated to version 3, using the changes made in the documentation:
```js
document.addEventListener("livewire:init", () => {
         $(function () {
             setTimeout(1000)
             var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
             var tooltipList = tooltipTriggerList.map(function (tooltipTriggerEl) {
                 return new bootstrap.Tooltip(tooltipTriggerEl)
             })
         })
});
```

Nothing else happens.

Anyone experiencing similar problems? I haven't found a solution yet to try to get around this.

**Reply by lrljoe** [reply]
Please check the docs around how LW3 interacts with your Javascript, it's really well documented!

**Reply by adityadees** [reply]
not sure, but the same issue is come from one people to one people, so it should not be celar doucmented!
